src/main/java/devcourse/board/domain/post/PostService.java

```diff
@@ -0,0 +1,73 @@
+package devcourse.board.domain.post;
+
+import devcourse.board.domain.member.MemberService;
+import devcourse.board.domain.member.model.Member;
+import devcourse.board.domain.post.model.Post;
+import devcourse.board.domain.post.model.PostRequest;
+import devcourse.board.domain.post.model.PostResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.persistence.EntityNotFoundException;
+import java.text.MessageFormat;
+import java.util.List;
+
+@Service
+@Transactional(readOnly = true)
+public class PostService {
+
+    private final MemberService memberService;
+
+    private final PostRepository postRepository;
+
+    public PostService(MemberService memberService, PostRepository postRepository) {
+        this.memberService = memberService;
+        this.postRepository = postRepository;
+    }
+
+    @Transactional
+    public Long createPost(PostRequest.CreationDto creationDto) {
+        Member member = memberService.findOne(creationDto.getMemberId());
+
+        Post post = PostRequest.toEntity(creationDto);
+        post.createPost(member);
+
+        postRepository.save(post);
+
+        return post.getId();
+    }
+
+    public PostResponse findOneAsDto(Long postId) {
+        return new PostResponse(findOne(postId));
+    }
```

Comment:
```suggestion
    public PostResponse findOne(Long postId) {
        return new PostResponse(findOne(postId));
    }
```
타입이 명시되는 언어라서 DTO를 반환하는걸 알고 있으니, 불필요한 메서드명은 제거해도 좋겠네요

Reply:
- `PostService` 에서 내부적으로 `Post` 엔티티를 찾는 [findOne 메서드](https://github.com/preferKim/project-jpa-board/blob/6579b05a293b91b966ee296825491aba88bc6b3b/src/main/java/devcourse/board/domain/post/PostService.java#L67)를 사용하고 있어 부득이하게 `findOneAsDto` 처럼 작명했었습니다 😂
- 현재 `findOneAsDto` 를 `findOne` 으로, 기존 [findOne 메서드](https://github.com/preferKim/project-jpa-board/blob/6579b05a293b91b966ee296825491aba88bc6b3b/src/main/java/devcourse/board/domain/post/PostService.java#L67)를  `findPostEntity` 로 변경했습니다. 여전히 '타입이 명시되는 언어'의 장점을 못 살리고 있는 것 같은데, 작명에 대해 조금 더 고민해보겠습니다.